resotocore/core/query/model.py

```diff
@@ -410,6 +417,29 @@ def change_variable(self, fn: Callable[[str], str]) -> Part:
             sort=[sort.change_variable(fn) for sort in self.sort],
         )
 
+    # ancestor.some_type.reported.prop -> MergeQuery
+    def merge_queries_for(self, property_paths: Iterable[str]) -> List[MergeQuery]:
+        def with_query_for(property_path: str) -> MergeQuery:
+            try:
+                assert is_ancestor_descendant(property_path)
+                anc_dec, kind, _ = property_path.split(".", 2)
+                direction = Direction.inbound if anc_dec == "ancestors" else Direction.outbound
+                navigation = Navigation(1, Navigation.Max, direction=direction)
+                subquery = Query([Part(IsTerm([kind])), Part(AllTerm(), navigation=navigation)])
+                return MergeQuery(f"{anc_dec}.{kind}", subquery)
+            except Exception as ex:
+                raise AttributeError(
+                    "The name of an ancestor variable has to follow the format: ancestor.<kind>.<path.to.variable>. "
```

Comment:
`ancestors` not `ancestor`

Reply:
Wrong correction messages are the worst - thanks for catching.